app/popup.js

```diff
@@ -21,55 +21,64 @@ function updateValue(e) {
 // When user click on "Add"
 function setWebsite() {
   let error = false;
-  chrome.storage.local.get(function (result) {
-    if (
-      typeof result['websites'] !== 'undefined' &&
-      result['websites'] instanceof Array
-    ) {
-      // Check if website is already in list
-      if (result['websites'].includes(log) === true) {
-        // If already in list, display error message
-        displayErrorMessage('Already in your list !');
-        document.getElementById('inputVal').focus();
-        error = true; // prevents closing popup
-      } else {
-        result['websites'].push(log);
-        for (const r in result['websites']) {
-          myURLsRedirect.push('*://*.' + result['websites'][r] + '/*');
+  log = log.trim();
+  // Check if input is not empty or filled with blanks and display error message
+  if (log.length === 0) {
```

Comment:
Did it in 2 steps to also prevent duplications because of blanks at beginning/end.